aspnet/web-api/overview/older-versions/using-web-api-1-with-entity-framework-5/using-web-api-with-entity-framework-part-3.md

```diff
@@ -21,18 +21,18 @@ In this section, we'll add a Web API controller that supports CRUD (create, read
 
 In Solution Explorer, right-click the Controllers folder. Select **Add** and then **Controller**.
 
-![](using-web-api-with-entity-framework-part-3/_static/image1.png)
+![Screenshot of the Solutions Explorer controllers menu. The add option is selected and Controller is highlighted.](using-web-api-with-entity-framework-part-3/_static/image1.png)
 
 In the **Add Controller** dialog, name the controller `AdminController`. Under **Template**, select &quot;API controller with read/write actions, using Entity Framework&quot;. Under **Model class**, select "Product (ProductStore.Models)". Under **Data Context**, select "&lt;New Data Context&gt;".
 
-![](using-web-api-with-entity-framework-part-3/_static/image2.png)
+![Screenshot of the Add Controller dialogue box. The data context class menu is open and new data context is highlighted.](using-web-api-with-entity-framework-part-3/_static/image2.png)
 
 > [!NOTE]
 > If the **Model class** drop-down does not show any model classes, make sure you compiled the project. Entity Framework uses reflection, so it needs the compiled assembly.
 
 Selecting "&lt;New Data Context&gt;" will open the **New Data Context** dialog. Name the data context `ProductStore.Models.OrdersContext`.
 
-![](using-web-api-with-entity-framework-part-3/_static/image3.png)
+![Screenshot of the new data context dialogue box. A textbox shows the name of the new data context typed in.](using-web-api-with-entity-framework-part-3/_static/image3.png)
```

Comment:
The alt-text uses the spelling "dialogue" but the text shows "dialog". Please be consistent.